pvlib/tests/test_tracking.py

```diff
@@ -462,17 +462,18 @@ def test_slope_aware_backtracking():
     """
     Test validation data set from https://www.nrel.gov/docs/fy20osti/76626.pdf
     """
+    # timestamps here are UTC instead of local standard time like in report
     expected_data = np.array(
-        [('2019-01-01T08:00-0500',  2.404287, 122.79177, -84.440, -10.899),
-         ('2019-01-01T09:00-0500', 11.263058, 133.288729, -72.604, -25.747),
-         ('2019-01-01T10:00-0500', 18.733558, 145.285552, -59.861, -59.861),
-         ('2019-01-01T11:00-0500', 24.109076, 158.939435, -45.578, -45.578),
-         ('2019-01-01T12:00-0500', 26.810735, 173.931802, -28.764, -28.764),
-         ('2019-01-01T13:00-0500', 26.482495, 189.371536, -8.475, -8.475),
-         ('2019-01-01T14:00-0500', 23.170447, 204.13681, 15.120, 15.120),
-         ('2019-01-01T15:00-0500', 17.296785, 217.446538, 39.562, 39.562),
-         ('2019-01-01T16:00-0500',  9.461862, 229.102218, 61.587, 32.339),
-         ('2019-01-01T17:00-0500',  0.524817, 239.330401, 79.530, 5.490)],
+        [('2019-01-01T13:00',  2.404287, 122.79177, -84.440, -10.899),
```

Comment:
I'm a bit surprised by this change. I'm sure it works but for style and clarity I would think including the timezone is better.

Reply:
Can we use `DataFrame`, `assert_series_equal`, and `assert_frame_equal` instead of `array` and `assert_allclose`? I think that would be more aligned with typical use cases and would solve the timezone problem.

Reply:
Some context here regarding numpy datetimes and timezones: https://numpy.org/doc/stable/release/1.11.0-notes.html#datetime64-changes